{"uuid":"36deac37-feac-457c-a5c2-dba359f692df","historyId":"e5b2cfc848f22c00bc826693e75e6f68","fullName":"org.SG.tests.DDT.LoginwithDDT.testLoginData","labels":[{"name":"package","value":"org.SG.tests.DDT.LoginwithDDT"},{"name":"testClass","value":"org.SG.tests.DDT.LoginwithDDT"},{"name":"testMethod","value":"testLoginData"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"APIAutomationFrameworkRA5X"},{"name":"subSuite","value":"org.SG.tests.DDT.LoginwithDDT"},{"name":"host","value":"DESKTOP-UGO2P2A"},{"name":"thread","value":"9348@DESKTOP-UGO2P2A.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testLoginData","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.RuntimeException: java.io.FileNotFoundException: D:\\Surekha\\ATB5X\\ATB5XRestAssured\\APIAutomationFrameworkRA5Xsrc\\main\\java\\org\\SG\\resources\\Testdata.xlsx (The system cannot find the path specified)","trace":"java.lang.RuntimeException: java.lang.RuntimeException: java.io.FileNotFoundException: D:\\Surekha\\ATB5X\\ATB5XRestAssured\\APIAutomationFrameworkRA5Xsrc\\main\\java\\org\\SG\\resources\\Testdata.xlsx (The system cannot find the path specified)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:812)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:737)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:915)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\r\n\tat org.testng.TestRunner.run(TestRunner.java:619)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.RuntimeException: java.io.FileNotFoundException: D:\\Surekha\\ATB5X\\ATB5XRestAssured\\APIAutomationFrameworkRA5Xsrc\\main\\java\\org\\SG\\resources\\Testdata.xlsx (The system cannot find the path specified)\r\n\tat org.SG.Utils.UtilExcel.getTestDataFromExcel(UtilExcel.java:28)\r\n\tat org.SG.Utils.UtilExcel.getData(UtilExcel.java:48)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\r\n\t... 24 more\r\nCaused by: java.io.FileNotFoundException: D:\\Surekha\\ATB5X\\ATB5XRestAssured\\APIAutomationFrameworkRA5Xsrc\\main\\java\\org\\SG\\resources\\Testdata.xlsx (The system cannot find the path specified)\r\n\tat java.base/java.io.FileInputStream.open0(Native Method)\r\n\tat java.base/java.io.FileInputStream.open(FileInputStream.java:213)\r\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)\r\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)\r\n\tat org.SG.Utils.UtilExcel.getTestDataFromExcel(UtilExcel.java:26)\r\n\t... 30 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1710126080508,"stop":1710126080514}